### DISCUSSION TITLE
I feel so dumb, but I can't find the settings

### DISCUSSION THREAD

**Discussion by SigmaTheDJ** [Anarios/return-youtube-dislike#532] [Q&A] [answered]
The add-on has just updated to version 3, which led to this screen popping up showing various settings that can be changed: -

https://i.imgur.com/rZOFSl6.jpg

However, I don't see anywhere to access them. In the "Manage extensions" screen in Firefox, if I click the 3 dots to the right of the add-on, it only says Remove, Report, and Manage. "Options" is not there.

Am I looking in the wrong place? Thanks!

**Comment by GeraltOfTrivia** [comment, marked as answer]
Click on the actual Return Youtube Dislike icon next to your url bar and then on the gear icon.
![εικόνα](https://user-images.githubusercontent.com/42317245/163609107-431e6c21-1beb-41be-bdca-825b7b46206e.png)

**Comment by Krit789** [comment]
![firefox_L37RHgyOwE](https://user-images.githubusercontent.com/68897623/163609940-2df2aefc-e72d-4892-954a-1b0e2a2ae1f5.gif)

**Comment by SigmaTheDJ** [comment]
Thanks! I must have removed that icon when I originally installed the add-on. Problem solved. :)

**Comment by MosquitoCoding** [comment]
Maybe someone should provide this information in the extension overview. I had to see this post as well to know where to find the settings
